code/Test_definitions/device-reachability-status-subscriptions.feature

```diff
@@ -147,67 +147,66 @@ Scenario: Receive notification for subscription-ends event on expiry
+  @reachability_status_subscription_15_invalid_protocol
+  Scenario: subscription creation with invalid protocol
     Given a valid subscription request body 
-    And  the request property "$.protocol" is not set to "HTTP"
+    And the request property "$.protocol" is not set to "HTTP"
     When the request "createDeviceReachabilityStatusSubscription" is sent
     Then the response property "$.status" is 400
     And the response property "$.code" is "INVALID_PROTOCOL"
     And the response property "$.message" contains a user friendly text
 
-@reachability_status_subscription_16_invalid_credential_type
-   Scenario: subscription creation with invalid credential type
+  @reachability_status_subscription_16_invalid_credential_type
+  Scenario: subscription creation with invalid credential type
     Given a valid subscription request body 
     And the request property "$.credentialType" is not "ACCESSTOKEN"
     When the request "createDeviceReachabilityStatusSubscription" is sent
     Then the response property "$.status" is 400
     And the response property "$.code" is "INVALID_CREDENTIAL"
     And the response property "$.message" contains a user friendly text
 	
-@reachability_status_subscription_17_invalid_access_token_type
-   Scenario: subscription creation with invalid access token type 
+  @reachability_status_subscription_17_invalid_access_token_type
+  Scenario: subscription creation with invalid access token type
     Given a valid subscription request body 
     And the request property "$.accessTokenType" is not "bearer"
     When the request "createDeviceReachabilityStatusSubscription" is sent
     Then the response property "$.status" is 400
     And the response property "$.code" is "INVALID_TOKEN" or "INVALID_ARGUMENT"
     And the response property "$.message" contains a user friendly text
 
-@reachability_status_subscription_18_invalid_credentials
-   Scenario: subscription creation with invalid credentials
+  @reachability_status_subscription_18_invalid_credentials
+  Scenario: subscription creation with invalid credentials
     Given a valid subscription request body 
     And header "Authorization" token is set to invalid credentials
     When the request "createDeviceReachabilityStatusSubscription" is sent
     Then the response property "$.status" is 401
     And the response property "$.code" is "UNAUTHENTICATED"
     And the response property "$.message" contains a user friendly text
 
-@reachability_status_subscription_19_invalid_inconsistent_access_token
-   Scenario: subscription creation with inconsistent access token for requested events subscription
-   # To test this, a token have to be obtained for a different device
+  @reachability_status_subscription_19_invalid_inconsistent_access_token
+  Scenario: subscription creation with inconsistent access token for requested events subscription
+    # To test this, a token have to be obtained for a different device
```

Comment:
Same issue that for the network-type-subscriptions - the SUBSCRIPTION_MISMATCH is for inconsistent access token for requested events subscription. For example, you had a 'consent' only for network-type subscription and you request roaming-change event type.